.env.sample

```diff
@@ -58,3 +58,4 @@ CIRRUS_FML_PATH=./feature_manifest/sample.yml
 CIRRUS_SENTRY_DSN=
 CIRRUS_INSTANCE_NAME=
 CIRRUS_ENV_NAME=test_instance_stage
+CIRRUS_GLEAN_MAX_EVENTS_BUFFER=20
```

Comment:
If we're setting it to 10 for prod, do we also want to set it to 10 here? (or 1, like test and integration-tests?)

Reply:
oh its just a random value, to show that we can adjust the values, we are not using this in our tests